packages/docusaurus-plugin-content-blog/src/index.js

```diff
@@ -131,29 +131,44 @@ class DocusaurusPluginContentBlog {
     }
 
     const blogTags = {};
+    const tagsPath = normalizeUrl([basePageUrl, 'tags']);
     blogPosts.forEach(blogPost => {
       const {tags} = blogPost.metadata;
       if (!tags || tags.length === 0) {
+        // TODO: Extract tags out into a separate plugin.
```

Comment:
What does this mean ?

Reply:
The tags code is very intertwined with the rest of the blogging code now. I think tags should be a transformer-like plugin that manipulates blog data instead of having it within the core blog plugin. The code is currently quite ugly and very hard to read now/